.github/workflows/build.yml

```
@@ -47,16 +48,111 @@ jobs:
+          echo "CURRENT_TAG is $CURRENT_TAG"
+          echo "MY_ENV_VAR=myValue" >> $GITHUB_ENV
+          Add-Content -Path ${env:GITHUB_ENV} -Value "CURRENT_TAG=$CURRENT_TAG"
+
+      - name: DEBUG Set environment variable in cmd
+        shell: cmd
+        run: |
+          printf "sleep_state2=purple" >> %GITHUB_ENV%
+
+      - name: DEBUG Set in bash
+        shell: bash
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:31: Double quote to prevent globbing and word splitting

(shellcheck)